test/services/database/challenge_repository_test.dart

```diff
@@ -128,6 +128,40 @@ void main() {
       expect(challenges.length, 1);
       expect(challenges.first.postId, posts[1].id); // the other post
     });
+
+    group("Regression tests", () {
+      // It seems that if you don't have any pastChallenges the mocked firestore
+      // will don't do any check on the query, so we need to add a pastChallenge to
+      // the user.
+      test("Challenge getter when more than 30 available posts works",
+          () async {
+        //Create a post that will be used as a completed past challenge.
```

Comment:
I meant to apply https://github.com/ProximaEPFL/proxima/pull/261#discussion_r1597566590 consistently everywhere in the file (and for future code too).

Reply:
You are completely right, I thought it was the only one and didn't double checked. 

I now double checked in both files, I think it should be good in [0ad5856](https://github.com/ProximaEPFL/proxima/pull/261/commits/0ad585698b97273f0daea96f1090d22f22398f39).

Thank you !